src/main/java/postrequsites/PostrequisiteHandler.java

```diff
@@ -0,0 +1,109 @@
+package postrequsites;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.json.JSONObject;
+
+import com.microsoft.azure.functions.ExecutionContext;
+import com.microsoft.azure.functions.HttpRequestMessage;
+import com.microsoft.azure.functions.HttpResponseMessage;
+
+import FuncTest.TestFunc.globals;
+import FuncTest.TestFunc.utils;
+import responses.TableResponse;
+
+public class PostrequisiteHandler {
+
+
+	public static HttpResponseMessage getPostrequisitesByNumber(JSONObject queryResult,
+			HttpRequestMessage<Optional<String>> s, ExecutionContext c) {
+		try {
+			JSONObject parameters = queryResult.getJSONObject("parameters");
+			List<String> requiredParameterNames = new ArrayList<>();
+			requiredParameterNames.add("courseNum");
```

Comment:
you could use Arrays.asList(T... x) instead of the new ArrayList<>

Also, you have a typo in the word "parameters" (in the error string)

Reply:
Fixed in latest commit.